spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -1025,14 +1025,13 @@ public O process(I payload) {
 }
 ----
 
-The `MANUAL` strategy indicates that the acknowledgment is managed by the user code. 
+The `MANUAL` strategy indicates that the acknowledgment is managed by the user code.
 The following snippet is particularly useful for processing messages that are also being sent to a destination, as the implementation must not invoke `ack` until after the outgoing message has been sent to the destination:
 
 [source, java]
 ----
 @Incoming("in")
 @Outgoing("out")
-@Acknowledgment(Acknowledgment.Strategy.MANUAL)
```

Comment:
I think this is better to change to PreProcessing to demonstrate how to set the undefault ack mode.

Reply:
So, I decided to add one more example explaining the `PRE_PROCESSING` strategy with Messages and re-add this line with a note that it's the default strategy.

Reply:
ok